src/main/java/com/moneylendingapp/advice/ApiAdviceHandler.java

```diff
@@ -0,0 +1,36 @@
+package com.moneylendingapp.advice;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.MethodParameter;
+import org.springframework.http.MediaType;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.http.server.ServerHttpRequest;
+import org.springframework.http.server.ServerHttpResponse;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseBodyAdvice;
+
+import java.time.Clock;
+import java.time.LocalDateTime;
+
+@RestControllerAdvice
+@RequiredArgsConstructor
+@Slf4j
+public class ApiAdviceHandler implements ResponseBodyAdvice<Object> {
+
+    private final Clock clock;
+
+    @Override
+    public boolean supports(MethodParameter returnType, Class<? extends HttpMessageConverter<?>> converterType) {
+        return true;
+    }
+
+    @Override
+    public Object beforeBodyWrite(Object body, MethodParameter returnType, MediaType selectedContentType, Class<? extends HttpMessageConverter<?>> selectedConverterType, ServerHttpRequest request, ServerHttpResponse response) {
```

Comment:
format this well. Try to avoid horizontal scrolling.

something like this

```
Object beforeBodyWrite (Object body, 
                                           MethodParameter returnType, 
                                           MediaType selectedContentType, 
                                           Class<? extends HttpMessageConverter<?>> selectedConverterType, 
                                           ServerHttpRequest request, ServerHttpResponse response) {
```